core/base/inc/RStipples.h

```diff
@@ -153,11 +153,16 @@ const unsigned char gStipples[26][32] = {
       0x80, 0x00, 0xc0, 0x00, 0xe0, 0x00, 0xf0, 0x00, 0xff, 0xf0, 0xff, 0xf0,
       0xfb, 0xf0, 0xf9, 0xf0, 0xf8, 0xf0, 0xf8, 0x70, 0xf8, 0x30, 0xff, 0xf0,
       0xff, 0xf8, 0xff, 0xfc, 0xff, 0xfe, 0xff, 0xff
-   }, //25
+   } //25
 };
 
 #else
 
+// Disable warning C4838: conversion from 'int' to 'const unsigned char' requires a narrowing conversion
+
+#pragma warning( push )
+#pragma warning( disable : 4838 )
```

Comment:
This doesn't create problems with other compilers?

Reply:
> This doesn't create problems with other compilers?

It is explicitly for windows - just check few lines before